python/swiftnav/almanac.pxd

```diff
@@ -7,7 +7,27 @@
 # EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
 # WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
 
```

Comment:
Yeah, we need to port some namespace documentation. I don't know how much of a slog it'll be to transition over docstrings, but we should probably do that for commonly used functions.